packages/@o3r/schematics/project.json

```diff
@@ -29,7 +29,17 @@
       "executor": "nx:run-script",
       "options": {
         "script": "build:builders"
-      }
+      },
+      "outputs": [
+        "{projectRoot}/dist/package.json",
```

Comment:
Shouldn't we have this change for all build:builders ?
because I think we have the same issue for @ama-sdk/schematics

Reply:
Actually no, they are not really touching the package.json usually